lib/connectors_sdk/base/http_call_wrapper.rb

```diff
@@ -99,8 +99,24 @@ def service_type
         self.class::SERVICE_TYPE
       end
 
+      def describe
+        {
+          :name => name,
+          :connection_requires_redirect => connection_requires_redirect,
+          :configurable_fields => configurable_fields
```

Comment:
What are configurable fields? Is it the connector settings?

Reply:
These are the fields the Enterprise Search UI should present to the user when configuring the connector.